globus_gsi_cert_utils_error.c:42: possible missing "," ?

I just tried to compile the package with new C compiler clang-14.
It said:

globus_gsi_cert_utils_error.c:42:11: warning: suspicious concatenation of string literals in an array initialization; did you mean to separate the elements with a comma? [-Wstring-concatenation]

Source code is

/* 7 */   "Proxy does not comply with proxy certificate standard"
/* 8 */   "Error determining certificate type"

--- a/common/source/configure.ac
+++ b/common/source/configure.ac
@@ -1,5 +1,5 @@
 AC_PREREQ([2.60])
-AC_INIT([globus_common], [18.12], [https://github.com/gridcf/gct/issues])
+AC_INIT([globus_common], [18.13], [https://github.com/gridcf/gct/issues])
 AC_CONFIG_MACRO_DIR([m4])
 AC_SUBST([MAJOR_VERSION], [${PACKAGE_VERSION%%.*}])
 AC_SUBST([MINOR_VERSION], [${PACKAGE_VERSION##*.}])

--- a/common/source/library/globus_options.c
+++ b/common/source/library/globus_options.c
@@ -104,17 +104,13 @@ globus_l_alphabetize_list(
     globus_options_entry_t *            e1 = low_datum;
     globus_options_entry_t *            e2 = high_datum;
 
-    if (e1->short_opt && !e2->short_opt)
+    if (!e2->short_opt)
     {
         return GLOBUS_TRUE;
     }
-    else if ((!e1->short_opt) && e2->short_opt)
+    else if (!e1->short_opt)
     {
         return GLOBUS_FALSE;
-    }
-    else if ((!e1->short_opt) && (!e2->short_opt))
-    {
-        return GLOBUS_TRUE;
     }
     else if (globus_libc_strncasecmp(e1->short_opt, e2->short_opt, strlen(e1->short_opt)) <= 0)
     {

--- a/gsi/cert_utils/source/configure.ac
+++ b/gsi/cert_utils/source/configure.ac
@@ -1,6 +1,6 @@
 AC_PREREQ([2.60])
 
-AC_INIT([globus_gsi_cert_utils], [10.8], [https://github.com/gridcf/gct/issues])
+AC_INIT([globus_gsi_cert_utils], [10.9], [https://github.com/gridcf/gct/issues])
 AC_CONFIG_MACRO_DIR([m4])
 AC_SUBST([MAJOR_VERSION], [${PACKAGE_VERSION%%.*}])
 AC_SUBST([MINOR_VERSION], [${PACKAGE_VERSION##*.}])

--- a/gsi/cert_utils/source/library/globus_gsi_cert_utils_error.c
+++ b/gsi/cert_utils/source/library/globus_gsi_cert_utils_error.c
@@ -38,7 +38,7 @@ globus_l_gsi_cert_utils_error_strings[GLOBUS_GSI_CERT_UTILS_ERROR_LAST] =
 /* 4 */   "Error adding Common Name to the certificate's subject",
 /* 5 */   "Out of memory",
 /* 6 */   "Error parsing subject name",
-/* 7 */   "Proxy does not comply with proxy certificate standard"
+/* 7 */   "Proxy does not comply with proxy certificate standard",
 /* 8 */   "Error determining certificate type"
 };
 

--- a/packaging/debian/globus-common/debian/changelog.in
+++ b/packaging/debian/globus-common/debian/changelog.in
@@ -1,3 +1,9 @@
+globus-common (18.13-1+gct.@distro@) @distro@; urgency=medium
+
+  * Simplify code
+
+ -- Mattias Ellert <mattias.ellert@physics.uu.se>  Mon, 11 Apr 2022 14:33:05 +0200
+
 globus-common (18.12-1+gct.@distro@) @distro@; urgency=medium
 
   * Fix some compiler and doxygen warnings

--- a/packaging/debian/globus-gsi-cert-utils/debian/changelog.in
+++ b/packaging/debian/globus-gsi-cert-utils/debian/changelog.in
@@ -1,3 +1,9 @@
+globus-gsi-cert-utils (10.9-1+gct.@distro@) @distro@; urgency=medium
+
+  * Add missing comma
+
+ -- Mattias Ellert <mattias.ellert@physics.uu.se>  Mon, 11 Apr 2022 14:34:06 +0200
+
 globus-gsi-cert-utils (10.8-1+gct.@distro@) @distro@; urgency=medium
 
   * Typo fixes

--- a/packaging/fedora/globus-common.spec
+++ b/packaging/fedora/globus-common.spec
@@ -3,7 +3,7 @@
 Name:		globus-common
 %global soname 0
 %global _name %(echo %{name} | tr - _)
-Version:	18.12
+Version:	18.13
 Release:	1%{?dist}
 Summary:	Grid Community Toolkit - Common Library
 
@@ -239,6 +239,9 @@ make %{?_smp_mflags} check VERBOSE=1 NO_EXTERNAL_NET=1
 %doc %{_pkgdocdir}/GLOBUS_LICENSE
 
 %changelog
+* Mon Apr 11 2022 Mattias Ellert <mattias.ellert@physics.uu.se> - 18.13-1
+- Simplify code
+
 * Wed Mar 09 2022 Mattias Ellert <mattias.ellert@physics.uu.se> - 18.12-1
 - Fix some compiler and doxygen warnings
 

--- a/packaging/fedora/globus-gsi-cert-utils.spec
+++ b/packaging/fedora/globus-gsi-cert-utils.spec
@@ -3,7 +3,7 @@
 Name:		globus-gsi-cert-utils
 %global soname 0
 %global _name %(echo %{name} | tr - _)
-Version:	10.8
+Version:	10.9
 Release:	1%{?dist}
 Summary:	Grid Community Toolkit - Globus GSI Cert Utils Library
 
@@ -180,6 +180,9 @@ make %{?_smp_mflags} check VERBOSE=1
 %doc %{_pkgdocdir}/GLOBUS_LICENSE
 
 %changelog
+* Mon Apr 11 2022 Mattias Ellert <mattias.ellert@physics.uu.se> - 10.9-1
+- Add missing comma
+
 * Fri Aug 20 2021 Mattias Ellert <mattias.ellert@physics.uu.se> - 10.8-1
 - Typo fixes
 

